HOW_TO_HANDLE_PULL_REQUESTS.md

```diff
@@ -6,9 +6,24 @@ If the pull request is from an employee of BigchainDB GmbH, then you can ignore
 
 If the pull request is from someone who is _not_ an employee of BigchainDB, then:
 
-* Have they agreed to the Individual Contributor Agreement in the past? (Troy, Greg, and others have a list.) If yes, then you can merge the PR and ignore the rest of this document.
-* Do they belong to a company or organization which agreed to the Entity Contributor Agreement in the past, and will they be contributing on behalf of that company or organization? (Troy, Greg, and others have a list.) If yes, then you can merge the PR and ignore the rest of this document.
-* Otherwise, go to the pull request in question and post a comment using this template:
+A. Have they agreed to the Individual Contributor Agreement in the past? There's a list of them in [a Google Spreadsheet that's accessible to all bigchaindb.com accounts](https://docs.google.com/spreadsheets/d/1VhekO6lgk1ZPx8dSjriucy4UinaU9pIdPQ5JXKcbD_Y/edit?usp=sharing). If yes, then you can merge the PR and ignore the rest of this document.
+
+B. Do they belong to a company or organization which agreed to the Entity Contributor Agreement in the past, and will they be contributing on behalf of that company or organization? (See the Google Spreadsheet link in A.) If yes, then you can merge the PR and ignore the rest of this document.
+
+C. Did they make a pull request to one of the bigchaindb repositories on GitHub (e.g. bigchaindb/bigchaindb)? If you're not sure, or you can't find one, then respond with an email of the form:
```

Comment:
I'm not sure what the intended context is here. These are instructions on how to respond to a PR from a non-bdb-employee. If we can't find a PR from them, why are we emailing them, and how do we have their email address in the first place?

Reply:
Sometimes people go to the form at https://www.bigchaindb.com/cla/ and fill it in (with their name, email address, etc.) without submitting a pull request first. It happens surprisingly often, so I figured I better add this documentation about how to handle that case.

Reply:
To minimize this edge case from happening, we could add a new required field to the CLA forms, requesting a link to a PR. If agreed, could someone add an issue or PR on `site` repo please?

Still cool to have those instructions for this in here, people could paste anything in that required field

Reply:
@kremalicious I got a CLA email from a guy recently, before he submitted a PR. I sent him this email, and he replied that he intends to submit a PR, but hadn't done so yet. I think that's okay. Therefore we don't need to add any new field to the CLA forms.